platform/android/java/lib/src/org/godotengine/godot/GodotLib.java

```diff
@@ -48,6 +48,7 @@
  */
 public class GodotLib {
 	static {
+		System.loadLibrary("System.Security.Cryptography.Native.Android");
```

Comment:
This needs to be guarded somehow so it only executes when there's a .NET/C# project, but I don't know how to do that in Java.

Alternatively, if there's a way to load this library from C++, I could move this to the Mono module. But this library implements `JNI_OnLoad` and I was unable to trigger that function when loading it from C++.

Reply:
The `loadLibrary` call should be moved to the `GodotApp.java` class because only the `app` module includes the `jar` dependency.

Reply:
Done.

Reply:
Each flavor specifies a dimension, and you can use that dimension to check for a specific flavor. See https://stackoverflow.com/a/18000544 for reference.

`BuildConfig` is the right file but it only gets generated after a build. You should be able to add references to it, which will resolve themselves at build time.